docs/architecture/management.md

```diff
@@ -0,0 +1,127 @@
+# Fabric Management
+
+This section focuses on operational aspects of Hedgehog Fabric, explaining available tools and typical workflows for administrators. It builds upon the architectural concepts explained in the [Overview](overview.md).
+
+The installation of a Hedgehog Fabric deployment is carried out using Fabricator (hhfab CLI). Once deployed, ongoing operations are managed via the [Kubernetes](https://kubernetes.io) CLI, [`kubectl`](https://kubernetes.io/docs/reference/kubectl/).
+
+In this workflow, the Kubernetes API Server processes the Fabric Custom Resources (CRs) and forwards them to the Fabric Controller. The Fabric Controller then translates these high-level network intents into concrete network configurations and manages their application to the underlying infrastructure. The Controller continuously monitors the state of the network resources, ensuring they match the desired state defined in the CRs.
+
+In this workflow, the user interacts with the Kubernetes API Server by creating or modifying Fabric Custom Resources (CRs). The Fabric Controller watches the Kubernetes API for changes to these CRs. Upon detecting a change, the Controller translates the high-level network intent into concrete configurations and applies them to the fabric switches.
+
+The Controller continuously reconciles the actual network state with the desired state declared in the CRs, ensuring consistency and reliability across the fabric.
+
+The diagram below illustrates the general workflow for fabric management:
+
+```mermaid
+graph TD
+
+%% Define the nodes
+User[User]
+Kubectl[kubectl]
+
+%% Control Node components
+subgraph ControlNode["Control Node"]
+  K8S[Kubernetes API Server]
+  FC[Fabric Controller]
+  K9s[K9s]
+  KubectlK3s[kubectl]
+  KubectlFabric["kubectl fabric"]
+end
+
+%% Define the relationships
+Kubectl -.->|Sends CRs and queries| K8S
+KubectlK3s -.->|Sends CRs and queries| K8S
+KubectlK3s -.->|Invokes| KubectlFabric
+User -.->|SSH Control| KubectlK3s
+User -.->|SSH Control| K9s
+User -.->|CLI| Kubectl
+KubectlFabric -->|Applies/inspects CRs| K8S
+K9s -->|Reads/writes CRs via API| K8S
+FC -->|Watches CRs| K8S
+```
+
+---
+
+## **Management Workflow Overview**
```

Comment:
I didn't go through the whole page, this is just a note in passing - I'd strongly recommend _against_ manually formatting all section/subsection headings in bold. The formatting from the theme should be enough already, no need to add manual formatting; if we decided it's not enough, this is something we should change by tuning the CSS for the whole website, for consistency.